Snowflake.Data/Core/RestRequester.cs

```diff
@@ -48,7 +48,19 @@ static internal RestRequester Instance
         public T Post<T>(IRestRequest request)
         {
             //Run synchronous in a new thread-pool task.
-            return Task.Run(async () => await PostAsync<T>(request, CancellationToken.None)).Result;
+            var task = Task.Run(async () => await PostAsync<T>(request, CancellationToken.None));
```

Comment:
These changes mainly just add logging around GetAsync/PostAsync exceptions.  If the goal was enhanced logging during failures it might be more explictly to simply add the logging in these methods (which would also benefit the ASYNC code paths -- not just the sync ones).